Blish HUD/Controls/TextInputBase.cs

```diff
@@ -583,14 +575,14 @@ public abstract class TextInputBase : Control {
         protected virtual void HandlePaste() {
             ClipboardUtil.WindowsClipboardService.GetTextAsync()
                          .ContinueWith((clipboardTask) => {
-                              if (!clipboardTask.IsFaulted) {
-                                  if (!string.IsNullOrEmpty(clipboardTask.Result)) {
-                                      Paste(clipboardTask.Result);
-                                  }
-                              } else {
+                             if (!clipboardTask.IsFaulted) {
+                                 if (!string.IsNullOrEmpty(clipboardTask.Result)) {
+                                     Paste(clipboardTask.Result);
+                                 }
```

Comment:
try reduce redundant paranthesis in single-line nested IF-statements using indent instead. Might look better.

Reply:
Not really part of this PR - appears to be an artifact of some minor formatting that got applied to the whole file.  👍